python/sgp4_scanner_stats.py

```diff
@@ -0,0 +1,165 @@
+'''
+Copyright (c) 2014-2016 Kartik Kumar, Dinamica Srl (me@kartikkumar.com)
+Copyright (c) 2014-2016 Abhishek Agrawal, Delft University of Technology
+                                          (abhishek.agrawal@protonmail.com)
+Distributed under the MIT License.
+See accompanying file LICENSE.md or copy at http://opensource.org/licenses/MIT
+'''
+
+# Set up modules and packages
+# I/O
+import csv
+import commentjson
+import json
+from pprint import pprint
+
+# SQL database
+import sqlite3
+
+# Numerical
+import numpy as np
+import pandas as pd
+
+# System
+import sys
+import time
+from os import path
+from tqdm import tqdm
+
+print ""
+print "---------------------------------------------------------------------------------"
+print "                                 D2D                                             "
+print "                                                                                 "
+print "         Copyright (c) 2016, K. Kumar (me@kartikkumar.com)                       "
+print "         Copyright (c) 2016, A. Agrawal (abhishek.agrawal@protonmail.com)        "
+print "---------------------------------------------------------------------------------"
+print ""
+
+# Start timer.
+start_time = time.time( )
+
+print ""
+print "******************************************************************"
+print "                          Input parameters                        "
+print "******************************************************************"
+print ""
+
+# Parse JSON configuration file
+# Raise exception if wrong number of inputs are provided to script
+if len(sys.argv) != 2:
+    raise Exception("Only provide a JSON config file as input!")
```

Comment:
Here tab is 4 spaces, later on it is 2 spaces. To be consistent throughout D2D 4 spaces have to be used.